decidim-core/app/queries/decidim/organization_participatory_processes.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module Decidim
+  # This query class returns the visible Participatory Processes given an
+  # Organization.
+  class OrganizationParticipatoryProcesses < Rectify::Query
```

Comment:
Sorry to put a comment after approving the PR but what about moving this to a `commands` folder like the others?